cms/templates/widgets/header.html

```diff
@@ -37,6 +37,9 @@ <h1 class="branding">
             certificates_url = ''
             if settings.FEATURES.get("CERTIFICATES_HTML_VIEW") and context_course.cert_html_view_enabled:
                 certificates_url = reverse('certificates_list_handler', kwargs={'course_key_string': six.text_type(course_key)})
+            exams_url = ''
+            if settings.FEATURES.get("EXAM_SETTINGS_HTML_VIEW"):
+                exams_url = settings.COURSE_AUTHORING_MICROFRONTEND_URL
```

Comment:
The URL to the course authoring MFE will have to be parameterized by the course key so that we should the correct settings to the user. Is it in the scope of this ticket to think about what kind of URL scheme we'd want for this and to implement it? Or do you think that should be done in a follow up ticket?

Reply:
I did think of this, but wasn't sure if it were in the scope of this ticket. I'd be more comfortable assigning it in a follow up ticket, but could also see how it fits within this ticket.

Reply:
Okay. That makes sense! Could you log a ticket for the follow up work?

Reply:
Sure!